integritybackend/c2patool.py

```diff
@@ -57,14 +57,13 @@ def run_claim_dump(self, asset_fullpath, claim_fullpath):
             Exception if errors are encountered
         """
         args = [
-            config.CLAIM_TOOL_PATH,
+            config.C2PA_TOOL_PATH,
             asset_fullpath,
-            "--dump_store",
```

Comment:
I'm unsure whether we want to use the `--detailed` flag here, which gives more detailed claim output. I think we don't, because the non-detailed output better aligns with the claim info we're putting in.

Here's an example using this image: https://user-images.githubusercontent.com/25111343/177398871-a58210eb-c42e-48b2-9666-f90ebf3df0d2.jpg

Non-detailed:
```
{
  "active_manifest": "adobe:urn:uuid:0b981d6e-68b7-46e8-90ec-3fb92588a280",
  "manifests": {
    "adobe:urn:uuid:0b981d6e-68b7-46e8-90ec-3fb92588a280": {
      "claim_generator": "Adobe Photoshop 23.3.1 (build 20220419.r.426 4d24a4c; mac; ContentCredentials 37f01a3)",
      "asset": {
        "title": "fake-news.jpg",
        "format": "image/jpeg",
        "instance_id": "xmp:iid:b46e7a29-0d87-45cc-9774-acfcb33e05ce"
      },
      "ingredients": [
        {
          "title": "AdobeStock_270923984.png",
          "format": "image/png",
          "document_id": "xmp:iid:fc0f2cbe-2b07-4244-9be9-0e3457a5ffd3",
          "instance_id": "xmp:iid:fc0f2cbe-2b07-4244-9be9-0e3457a5ffd3",
          "is_parent": true,
          "validation_status": [],
          "metadata": {
            "dateTime": "2022-05-06T20:04:49.410Z"
          }
        },
        {
          "title": "social-media.png",
          "format": "image/png",
          "document_id": "xmp:iid:f15e544f-045f-4306-8db4-ef250dde7554",
          "instance_id": "xmp:iid:f15e544f-045f-4306-8db4-ef250dde7554",
          "validation_status": [],
          "metadata": {
            "dateTime": "2022-05-06T20:06:24.863Z"
          }
        }
      ],
      "credentials": [],
      "assertions": [
        {
          "label": "adobe.beta",
          "data": {
            "version": "0.12.5"
          }
        },
        {
          "label": "adobe.dictionary",
          "data": {
            "url": "https://cai-assertions.adobe.com/photoshop/dictionary.json"
          }
        },
        {
          "label": "stds.schema-org.CreativeWork",
          "data": {
            "@context": "https://schema.org",
            "@type": "CreativeWork",
            "author": []
          },
          "kind": "Json"
        },
        {
          "label": "c2pa.actions",
          "data": {
            "actions": [
              {
                "action": "c2pa.edited",
                "parameters": {
                  "name": "open"
                }
              },
              {
                "action": "c2pa.edited",
                "parameters": {
                  "name": "desaturate"
                }
              },
              {
                "action": "c2pa.edited",
                "parameters": {
                  "name": "image_size"
                }
              },
              {
                "action": "c2pa.edited",
                "parameters": {
                  "name": "place_image"
                }
              },
              {
                "action": "c2pa.edited",
                "parameters": {
                  "name": "create_clipping_mask"
                }
              },
              {
                "action": "c2pa.edited",
                "parameters": {
                  "name": "brush_tool"
                }
              },
              {
                "action": "c2pa.edited",
                "parameters": {
                  "name": "stroke"
                }
              },
              {
                "action": "c2pa.edited",
                "parameters": {
                  "name": "master_opacity_change"
                }
              },
              {
                "action": "c2pa.edited",
                "parameters": {
                  "name": "layer_visibility"
                }
              }
            ],
            "metadata": {
              "dateTime": "2022-05-06T20:04:49.373Z",
              "reviewRatings": []
            }
          }
        },
        {
          "label": "c2pa.hash.data",
          "data": {
            "alg": "sha256",
            "exclusions": [
              {
                "length": 1296038,
                "start": 2483
              }
            ],
            "hash": "uTE3HOp1Hx8g/uCB+IdlpecCGPRw5ihQFxvsYuBb5yg=",
            "name": "jumbf manifest",
            "pad": "<omitted>"
          }
        }
      ],
      "signature_info": {
        "issuer": "Adobe Inc.",
        "time": "2022-05-06T20:09:38+00:00"
      }
    }
  }
}
```

Detailed:

```
{
  "active_manifest": "adobe:urn:uuid:0b981d6e-68b7-46e8-90ec-3fb92588a280",
  "manifests": {
    "adobe:urn:uuid:0b981d6e-68b7-46e8-90ec-3fb92588a280": {
      "claim": {
        "alg": "sha256",
        "assertions": [
          {
            "hash": "oYBuHO6kdwimVi0dlyUUY/l/QH7ywKpptFaF2bHIO/w=",
            "url": "self#jumbf=c2pa.assertions/c2pa.thumbnail.claim.jpeg"
          },
          {
            "hash": "tBdofoM7zlMEv3OyFjgq3PjeS9jBlpNB4Zxa5WTP/eg=",
            "url": "self#jumbf=c2pa.assertions/c2pa.thumbnail.ingredient.png"
          },
          {
            "hash": "JNq089RX5Do1woqBGOtK7S9TRhgsy0fRhOnGV/ACb7c=",
            "url": "self#jumbf=c2pa.assertions/c2pa.ingredient"
          },
          {
            "hash": "Yv/8pWVHI4yTb+kq7vT7W6HPqxjzmH/VXwKi5VykIvw=",
            "url": "self#jumbf=c2pa.assertions/c2pa.thumbnail.ingredient__1.png"
          },
          {
            "hash": "7vUiNys4IIa4E2WOz8x3sHpQESZRMtko64UoRYbWEkU=",
            "url": "self#jumbf=c2pa.assertions/c2pa.ingredient__1"
          },
          {
            "hash": "I4w08CRpaJOfnqz1SXBjJgdO96G3FzgUDYJyw4TCatQ=",
            "url": "self#jumbf=c2pa.assertions/adobe.beta"
          },
          {
            "hash": "PM0ve0P7crI+hKZhXK0FIoPiLOR+DnFpUtFKek/ugA0=",
            "url": "self#jumbf=c2pa.assertions/adobe.dictionary"
          },
          {
            "hash": "IZXcq7Vjh0n8rIJ3xt/fymDnnHOxIL7rWU9tp9UqXd0=",
            "url": "self#jumbf=c2pa.assertions/stds.schema-org.CreativeWork"
          },
          {
            "hash": "tPr46vRkthTyiN3MT3qSGce+XDjuQD6XnBfpaa415F4=",
            "url": "self#jumbf=c2pa.assertions/c2pa.actions"
          },
          {
            "hash": "gUicKeZCoD0r9f5jgO8gZllBNn+HEYaraQoAOvpA23w=",
            "url": "self#jumbf=c2pa.assertions/c2pa.hash.data"
          }
        ],
        "claim_generator": "Adobe Photoshop 23.3.1 (build 20220419.r.426 4d24a4c; mac; ContentCredentials 37f01a3)",
        "dc:format": "image/jpeg",
        "dc:title": "fake-news.jpg",
        "instanceID": "xmp:iid:b46e7a29-0d87-45cc-9774-acfcb33e05ce",
        "signature": "self#jumbf=c2pa.signature"
      },
      "assertion_store": {
        "c2pa.ingredient__1": {
          "dc:format": "image/png",
          "dc:title": "social-media.png",
          "documentID": "xmp:iid:f15e544f-045f-4306-8db4-ef250dde7554",
          "instanceID": "xmp:iid:f15e544f-045f-4306-8db4-ef250dde7554",
          "metadata": {
            "dateTime": "2022-05-06T20:06:24.863Z"
          },
          "relationship": "componentOf",
          "thumbnail": {
            "hash": "Yv/8pWVHI4yTb+kq7vT7W6HPqxjzmH/VXwKi5VykIvw=",
            "url": "self#jumbf=c2pa.assertions/c2pa.thumbnail.ingredient__1.png"
          },
          "validationStatus": []
        },
        "c2pa.thumbnail.claim.jpeg": "<omitted> len = 87984",
        "stds.schema-org.CreativeWork": {
          "@context": "https://schema.org",
          "@type": "CreativeWork",
          "author": []
        },
        "adobe.dictionary": {
          "url": "https://cai-assertions.adobe.com/photoshop/dictionary.json"
        },
        "adobe.beta": {
          "version": "0.12.5"
        },
        "c2pa.hash.data": {
          "alg": "sha256",
          "exclusions": [
            {
              "length": 1296038,
              "start": 2483
            }
          ],
          "hash": "uTE3HOp1Hx8g/uCB+IdlpecCGPRw5ihQFxvsYuBb5yg=",
          "name": "jumbf manifest",
          "pad": "<omitted>"
        },
        "c2pa.thumbnail.ingredient.png": "<omitted> len = 927193",
        "c2pa.actions": {
          "actions": [
            {
              "action": "c2pa.edited",
              "parameters": {
                "name": "open"
              }
            },
            {
              "action": "c2pa.edited",
              "parameters": {
                "name": "desaturate"
              }
            },
            {
              "action": "c2pa.edited",
              "parameters": {
                "name": "image_size"
              }
            },
            {
              "action": "c2pa.edited",
              "parameters": {
                "name": "place_image"
              }
            },
            {
              "action": "c2pa.edited",
              "parameters": {
                "name": "create_clipping_mask"
              }
            },
            {
              "action": "c2pa.edited",
              "parameters": {
                "name": "brush_tool"
              }
            },
            {
              "action": "c2pa.edited",
              "parameters": {
                "name": "stroke"
              }
            },
            {
              "action": "c2pa.edited",
              "parameters": {
                "name": "master_opacity_change"
              }
            },
            {
              "action": "c2pa.edited",
              "parameters": {
                "name": "layer_visibility"
              }
            }
          ],
          "metadata": {
            "dateTime": "2022-05-06T20:04:49.373Z",
            "reviewRatings": []
          }
        },
        "c2pa.ingredient": {
          "dc:format": "image/png",
          "dc:title": "AdobeStock_270923984.png",
          "documentID": "xmp:iid:fc0f2cbe-2b07-4244-9be9-0e3457a5ffd3",
          "instanceID": "xmp:iid:fc0f2cbe-2b07-4244-9be9-0e3457a5ffd3",
          "metadata": {
            "dateTime": "2022-05-06T20:04:49.410Z"
          },
          "relationship": "parentOf",
          "thumbnail": {
            "hash": "tBdofoM7zlMEv3OyFjgq3PjeS9jBlpNB4Zxa5WTP/eg=",
            "url": "self#jumbf=c2pa.assertions/c2pa.thumbnail.ingredient.png"
          },
          "validationStatus": []
        },
        "c2pa.thumbnail.ingredient__1.png": "<omitted> len = 266537"
      },
      "signature": {
        "alg": "ps256",
        "issuer": "Adobe Inc.",
        "time": "2022-05-06T20:09:38+00:00"
      }
    }
  },
  "validation_status": [
    {
      "code": "claimSignature.validated",
      "url": "self#jumbf=/c2pa/adobe:urn:uuid:0b981d6e-68b7-46e8-90ec-3fb92588a280/c2pa.signature",
      "explanation": "claim signature valid"
    },
    {
      "code": "assertion.hashedURI.match",
      "url": "self#jumbf=c2pa.assertions/c2pa.thumbnail.claim.jpeg",
      "explanation": "hashed uri matched: self#jumbf=c2pa.assertions/c2pa.thumbnail.claim.jpeg"
    },
    {
      "code": "assertion.hashedURI.match",
      "url": "self#jumbf=c2pa.assertions/c2pa.thumbnail.ingredient.png",
      "explanation": "hashed uri matched: self#jumbf=c2pa.assertions/c2pa.thumbnail.ingredient.png"
    },
    {
      "code": "assertion.hashedURI.match",
      "url": "self#jumbf=c2pa.assertions/c2pa.ingredient",
      "explanation": "hashed uri matched: self#jumbf=c2pa.assertions/c2pa.ingredient"
    },
    {
      "code": "assertion.hashedURI.match",
      "url": "self#jumbf=c2pa.assertions/c2pa.thumbnail.ingredient__1.png",
      "explanation": "hashed uri matched: self#jumbf=c2pa.assertions/c2pa.thumbnail.ingredient__1.png"
    },
    {
      "code": "assertion.hashedURI.match",
      "url": "self#jumbf=c2pa.assertions/c2pa.ingredient__1",
      "explanation": "hashed uri matched: self#jumbf=c2pa.assertions/c2pa.ingredient__1"
    },
    {
      "code": "assertion.hashedURI.match",
      "url": "self#jumbf=c2pa.assertions/adobe.beta",
      "explanation": "hashed uri matched: self#jumbf=c2pa.assertions/adobe.beta"
    },
    {
      "code": "assertion.hashedURI.match",
      "url": "self#jumbf=c2pa.assertions/adobe.dictionary",
      "explanation": "hashed uri matched: self#jumbf=c2pa.assertions/adobe.dictionary"
    },
    {
      "code": "assertion.hashedURI.match",
      "url": "self#jumbf=c2pa.assertions/stds.schema-org.CreativeWork",
      "explanation": "hashed uri matched: self#jumbf=c2pa.assertions/stds.schema-org.CreativeWork"
    },
    {
      "code": "assertion.hashedURI.match",
      "url": "self#jumbf=c2pa.assertions/c2pa.actions",
      "explanation": "hashed uri matched: self#jumbf=c2pa.assertions/c2pa.actions"
    },
    {
      "code": "assertion.hashedURI.match",
      "url": "self#jumbf=c2pa.assertions/c2pa.hash.data",
      "explanation": "hashed uri matched: self#jumbf=c2pa.assertions/c2pa.hash.data"
    },
    {
      "code": "assertion.dataHash.match",
      "url": "self#jumbf=/c2pa/adobe:urn:uuid:0b981d6e-68b7-46e8-90ec-3fb92588a280/c2pa.assertions/c2pa.hash.data",
      "explanation": "data hash valid"
    }
  ]
}
```
```